src/servlet/MyServlet.java

```diff
@@ -0,0 +1,18 @@
+package servlet;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+
+    public class MyServlet extends HttpServlet {
+        @Override
+        protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+            try(OutputStreamWriter writer=new OutputStreamWriter(resp.getOutputStream())){
```

Comment:
(cosmetic) Space between `try` and `(` is needed. Spaceas around `=` are needed throughout the project. These are some codestyle standards